src/main/resources/static/vulnerableApp.js

```diff
@@ -321,3 +321,24 @@ function _addingEventListenerToShowHideHelpButton(vulnerableAppEndPointData) {
     document.getElementById("aboutContainer").scrollIntoView(true);
   });
 })();
+
+
+document.getElementById("learnAndPracticeBtn").addEventListener("click", pushToVulnerabilities);
+let previousUrl;
+
+function pushToVulnerabilities() {
+  let b = document.getElementById("learnAndPracticeBtn");
+  previousUrl = './learnVulnerabilities';
```

Comment:
we can have URL's with the element id's. WDYT ?

Reply:
I think it's a good idea.

Reply:
also 
Hi @ShareACoke1 ,

I had looked at the code, actually if we can do the same with opening the URL directly in browser then it would be great. I was thinking that we can parse the url, find the id's and click on the elements with those id's. this way we can achieve the full flow.

WDYT ?
Also i was planning to look at the router logics in frameworks so that we can do the same.

thanks,
Karan